src/common/feedGenerator.ts

```diff
@@ -167,13 +167,20 @@ export const feedToFilters = async (
     membershipsByHide.hide.push(WATERCOOLER_ID);
   }
 
+  const blockedContentCuration = settings
+    .filter(
+      (setting) => setting.group === 'content_curation' && setting.options.type,
+    )
+    .map((setting) => setting.options.type);
```

Comment:
You can do it in one go with `reduce`. Just so feed is critical and we probably want to be optimized here?

Reply:
Ah yeah good point I just lazy copy pasted another one above. 😅
Can refactor both tbh